float64 interval_time
int16 motor1_rpm
int32 motor1_pos
int16 motor2_rpm
int32 motor2_pos
uint8 motor1_state
uint8 motor2_state
